### PR TITLE
ImageCard: add selectable treatment to subtle cards

### PR DESCRIPTION
**Pull request by michael-iden**
**This change:** (check at least one)

- [X] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [ ] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [X] No

**Is the:** (complete all)

- [X] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [ ] Test suite(s) passing?
- [ ] Code coverage maximal?
- [ ] Changeset added?
- [ ] Component status page up to date?

**What does this change address?**
If the card has `subtle` set to `true`, the metadata shows on hover but the checkbox would be covered up and overlapping. This moves the checkbox to be within the metadata hover overlay for subtle cards, above the title.

**How does this change work?**
- Changes when we put the checkbox within the DOM for subtle cards
- Add classes and update when they are applied for proper positioning

**Additional context**
The card is getting somewhat difficult to maintain with the list of variants and also additional boolean attributes that can also be treated like a variant. `subtle` also used in two different contexts within in the component. I'd be open to a discussion around to both clean up the interface/implementation and work with design to limit the number of card experiences we support

image

### CODE REVIEW

**Review by daneah — APPROVED** [review]
This looks great. I see your concern about the complexity of the component; I can raise this in the next Pharos operations discussion.

File: packages/pharos/src/components/image-card/pharos-image-card.ts [hunk 1]
```diff
@@ -254,7 +254,9 @@ export class PharosImageCard extends ScopedRegistryMixin(FocusMixin(PharosElemen
   private _renderHoverMetadata(): TemplateResult | typeof nothing {
     return this.subtle
       ? html`<div class="card__metadata--hover">
-          <strong class="card__title--hover">${this.title}</strong><slot name="metadata"></slot>
+          ${this._renderCheckbox()}
+          <strong class="card__title--hover">${this.title}</strong>
+          <slot name="metadata"></slot>
```
Comment (jialin-he) [inline packages/pharos/src/components/image-card/pharos-image-card.ts:259]: Do we need to hide the metadata when the card is selected?

Comment (michael-iden) [reply packages/pharos/src/components/image-card/pharos-image-card.ts:259]: AFAIK yes, although I haven't had a chance to run this by Lori yet to ensure its looking as she expected

**Comment by daneah** [comment]
@michael-iden you might already be on it, but it looks like the latest commits broke some image displays and heading styles.

File: packages/pharos/src/components/image-card/pharos-image-card.wc.stories.mdx [hunk 2]
```diff
@@ -177,6 +177,70 @@ export const Selectable = () => html`<pharos-layout tag="ol" style="margin: 1rem
   </li>
 </pharos-layout>`;
 
+export const SubtleSelectable = () => html`<pharos-layout tag="ol" style="margin: 1rem 0">
```
Comment (daneah) [inline packages/pharos/src/components/image-card/pharos-image-card.wc.stories.mdx:180]: This appears to be an exact copy of the existing *Selectable* story which already includes the subtle selectable card—might we not need this?

File: packages/pharos/src/components/image-card/pharos-image-card.ts (hunk 1 above)
Comment (michael-iden) [reply packages/pharos/src/components/image-card/pharos-image-card.ts:259]: Did give Lori a moment to look a this and you are right. We do now hide the metadata on selection

https://user-images.githubusercontent.com/13315416/191797133-d84c0f24-8ac7-49ee-9918-c0669d15dbff.mov

File: packages/pharos/src/components/image-card/pharos-image-card.wc.stories.mdx (hunk 2 above)
Comment (michael-iden) [reply packages/pharos/src/components/image-card/pharos-image-card.wc.stories.mdx:180]: It's a little different in that this block has the `subtle="true"` on all the card elements, whereas the `Selectable` story does not. But I think that reinforces the confusion over the # of variations we have here and the difficulty supporting them.

image

**Review by jialin-he — APPROVED** [review]

**Pull request merged**